### PR TITLE
Added Kurtosis function

### LINKED ISSUE #16360: Add kurtosis function in stats
Kurtosis is an essential measure in Statistics. It is all about the tails of the distribution. It is used to describe the extreme values in one versus the other tail. It is actually the measure of outliers present in the distribution[[Medium]](https://codeburst.io/2-important-statistics-terms-you-need-to-know-in-data-science-skewness-and-kurtosis-388fef94eeaa)[[Wikipedia](https://en.wikipedia.org/wiki/Kurtosis#Interpretation)]

### PR DESCRIPTION
**Pull request by ritesh99rakesh**
#### References to other Issues or PRs

Closes #16361 
Fixes #16360

#### Brief description of what is fixed or changed
Added kurtosis function in `sympy/stats/rv_interface.py`
and added tests for different distributions.

#### Other comments
I deleted my previous fork of `sympy` and hence reopening this PR.
ping @sidhantnagpal @oscargus 

#### Release Notes

- stats
    - added `kurtosis` method

### CODE REVIEW

File: sympy/stats/rv_interface.py [hunk 1]
```diff
@@ -248,7 +248,36 @@ def skewness(X, condition=None, **kwargs):
     >>> skewness(Y)
     2
     """
-    return smoment(X, 3, condition, **kwargs)
+    return smoment(X, 3, condition=condition, **kwargs)
+
+def kurtosis(X, condition=None, **kwargs):
+    """
+    Measure of the tailedness (outliers) of the probability distribution.
```
Comment (smichr) [inline sympy/stats/rv_interface.py]:
```suggestion
    Characterizes the tails/outliers of a probability distribution.
```

Comment (smichr) [reply sympy/stats/rv_interface.py]: Please add a description of how to use `condition`.

File: sympy/stats/rv_interface.py [hunk 2]
```diff
@@ -228,12 +228,18 @@ def smoment(X, n, condition=None, **kwargs):
...
 
     Positive skew indicates that most of the values lie to the right of
-    the mean
+    the mean.
 
-    skewness(X) = E( ((X - E(X))/sigma)**3 )
+    skewness(X) = E(((X - E(X))/sigma)**3)
+
+    Parameter
+    ==========
```
Comment (czgdp1807) [inline sympy/stats/rv_interface.py]: Remove extra `=`

File: sympy/stats/rv_interface.py [hunk 3]
```diff
@@ -243,12 +249,53 @@ def skewness(X, condition=None, **kwargs):
...
+    (-4/pi - 12/pi**2 + 3)/(1 - 2/pi)**2
+
+    >>> rate = Symbol('lamda', positive=True, real=True, finite=True)
+    >>> Y = Exponential('Y', rate)
+    >>> kurtosis(Y)
+    9
+
+    References
+    ==========
+    .. [1] https://en.wikipedia.org/wiki/Kurtosis
```
Comment (czgdp1807) [inline sympy/stats/rv_interface.py]: A wolfram link would be good to add

File: sympy/stats/rv_interface.py (hunk 2 above)
Comment (ritesh99rakesh) [reply sympy/stats/rv_interface.py]: Done. Thanks!

File: sympy/stats/rv_interface.py (hunk 3 above)
Comment (ritesh99rakesh) [reply sympy/stats/rv_interface.py]: Done.

File: sympy/stats/rv_interface.py [hunk 4]
```diff
@@ -228,12 +228,18 @@ def smoment(X, n, condition=None, **kwargs):
...
+    Measure of the asymmetry of the probability distribution.
 
     Positive skew indicates that most of the values lie to the right of
-    the mean
+    the mean.
 
-    skewness(X) = E( ((X - E(X))/sigma)**3 )
+    skewness(X) = E(((X - E(X))/sigma)**3)
+
+    Parameter
```
Comment (sidhantnagpal) [inline sympy/stats/rv_interface.py]: The header is named "Parameters" (plural).

**Comment by ritesh99rakesh** [comment]
@sidhantnagpal Please review this.

**Comment by sidhantnagpal** [comment]
Looks good to me. Thanks.

**Pull request merged**